### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Nathancooke7**

### CODE REVIEW

**Comment by bstiq** [comment]
Pinging @0xcharly to take a look, but lgtm

File: keyboards/bastardkb/charybdis/readme.md [hunk 1]
```diff
@@ -114,6 +143,20 @@ This behavior can be further customized with the following defines:
...
+```c
+#define CHARYBDIS_POINTER_ACCELERATION_ENABLE
+```
+
+The acceleration factor can be further tune _via_ the `CHARYBDIS_POINTER_ACCELERATION_FACTOR`:
+
+```c
+#define CHARYBDIS_POINTER_ACCELERATION_FACTOR 24
+```
+
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/readme.md:159]: This was removed in https://github.com/Bastardkb/bastardkb-qmk/commit/96c052e161e02b7e2c56fd4e047f69438cc3658c.  Did you mean to add it back?

File: keyboards/bastardkb/charybdis/readme.md [hunk 2]
```diff
@@ -18,32 +18,61 @@ Check out the [keyboard build guides](https://docs.bastardkb.com) for the Charyb
...
 ### Charybdis (4x6)
 
 The `default` keymap is inspired from the original [Dactyl Manuform](../../handwired/dactyl_manuform) layout.
 
+* For Elite-C v1 shield
+
 ```shell
-qmk compile -kb bastardkb/charybdis/4x6 -km default
+qmk compile -kb bastardkb/charybdis/4x6/1.x/elitec -km default
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/readme.md:30]: This changed recently after updating the QMK's `master` branch.  The dot (`.`) is no longer a valid character in the keyboard path.  It should be:

- `bastardkb/charybdis/3x5/v1/elitec`
- `bastardkb/charybdis/3x5/v2/elitec`
- `bastardkb/charybdis/4x6/v1/elitec`
- `bastardkb/charybdis/4x6/v2/elitec`

File: keyboards/bastardkb/charybdis/readme.md [hunk 3]
```diff
@@ -18,32 +18,61 @@ Check out the [keyboard build guides](https://docs.bastardkb.com) for the Charyb
...
+```
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/readme.md:31]: This needs to be indented to be part of the bullet point I believe.

**Review by 0xcharly — CHANGES_REQUESTED** [review]
Thanks for taking the time to contribute!

File: keyboards/bastardkb/charybdis/readme.md (hunk 1 above)
Comment (Nathancooke7) [reply keyboards/bastardkb/charybdis/readme.md:159]: no I must have based off an outdated branch, I'll remove this.

File: keyboards/bastardkb/charybdis/readme.md (hunk 2 above)
Comment (Nathancooke7) [reply keyboards/bastardkb/charybdis/readme.md:30]: updated

File: keyboards/bastardkb/charybdis/readme.md (hunk 3 above)
Comment (Nathancooke7) [reply keyboards/bastardkb/charybdis/readme.md:31]: good point, updated

**Comment by Nathancooke7** [comment]
@0xcharly are there any other changes I should make?

File: keyboards/bastardkb/charybdis/readme.md [hunk 4]
```diff
@@ -18,32 +18,63 @@ Check out the [keyboard build guides](https://docs.bastardkb.com) for the Charyb
...
 ```shell
-qmk compile -kb bastardkb/charybdis/4x6 -km default
+	qmk compile -kb bastardkb/charybdis/4x6/v2/elitec -km default
 ```
 
 Check out the `via` layout if you're looking for VIA support:
 
+* For Elite-C v1 shield
+
+```shell
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/readme.md:42]: I think the entire block needs to be indented by 4 spaces, not just the content of the code-block.  In the preview, the code lock should start at the same position as the first letter of the bullet point list.

Comment (Nathancooke7) [reply keyboards/bastardkb/charybdis/readme.md]: I switched it to be a table. I think that's clearer

**Pull request merged**